helix-core/src/main/resources/soft-constraint-weight.properties

```diff
@@ -17,8 +17,10 @@
 # under the License.
 #
 
-PartitionMovementConstraint=1f
-InstancePartitionsCountConstraint=0.3f
-ResourcePartitionAntiAffinityConstraint=0.1f
-ResourceTopStateAntiAffinityConstraint=0.1f
-MaxCapacityUsageInstanceConstraint=0.5f
\ No newline at end of file
+# Define the constraint weights for the WAGED rebalancer in this file.
```

Comment:
Why not use the property file and manually override the hashmap value?

Reply:
We just need one set of default values. So the file can be defaulted to be empty.
I'm also fine with putting the default values to the resource file. But it would be hard to test and play with the code while we are developing. I mean either way works. Just don't keep both.